src/main/java/ti4/image/PlayerAreaGenerator.java

```diff
@@ -1746,6 +1746,10 @@ private int drawPlanetInfo(Player player, String planetName, int x, int y, int d
 
         try {
             Planet planet = planetsInfo.get(planetName);
+            if (planet == null) {
+                BotLogger.error(new BotLogger.LogMessageOrigin(player), "Planet " + planetName + " not found in game " + game.getName());
+                return deltaX;
+            }
```

Comment:
I *think* this will help debugging situations where the planet name can't print. If nothing else, it will help distinguish between this case and other exceptions in this block.